src/html/profile.html

```diff
@@ -0,0 +1,442 @@
+                    <span class="material-symbols-outlined text-secondary">
+                      person_remove
+                    </span>
+                  </div>
+                </li>
+              </ul>
+              <ul
+                class="container bg-body card"
+                id="suggested-contacts-desktop"
+              >
+                <li class="list-unstyled">
+                  <h2 class="h5 text-primary">suggestions</h2>
+                </li>
+                <li class="row d-flex py-3 list-unstyled">
+                  <div class="col-xl-2 col-lg-3 col-2">
+                    <img
+                      src="../img/rick-gebhardt-3faX71BaNe0-unsplash.png"
+                      class="img-fluid"
+                      alt="user placeholder"
+                    />
+                  </div>
+                  <div class="col-6">rick wandel</div>
+                  <div class="col-2">
+                    <span class="material-symbols-outlined text-secondary">
+                      person_add
+                    </span>
+                  </div>
+                </li>
+                <li class="row d-flex py-3">
+                  <div class="col-xl-2 col-lg-3 col-2">
+                    <img
+                      src="../img/david-todd-mccarty-VGLImIsgs9A-unsplash.png"
+                      class="img-fluid"
+                      alt="user placeholder"
+                    />
+                  </div>
+                  <div class="col-6">marcia mccarthy</div>
+                  <div class="col-2">
+                    <span class="material-symbols-outlined text-secondary">
+                      person_add
+                    </span>
+                  </div>
+                </li>
+                <li class="row d-flex py-3">
+                  <div class="col-xl-2 col-lg-3 col-2">
+                    <img
+                      src="../img/jacob-townsend-4OzZru1zewE-unsplash.png"
+                      class="img-fluid"
+                      alt="user placeholder"
+                    />
+                  </div>
+                  <div class="col-6">adam farmer</div>
+                  <div class="col-2">
+                    <span class="material-symbols-outlined text-secondary">
+                      person_add
+                    </span>
+                  </div>
+                </li>
+              </ul>
+            </div>
+          </div>
+          <div class="col-lg-9">
+            <div class="col col-lg-12">
+              <div class="container py-5 bg-light d-none d-lg-block">
+                <div class="card my-1 border-0">
+                  <form action="">
+                    <div class="form-floating">
+                      <textarea
+                        name=""
+                        id="user-post-desktop"
+                        class="form-control w-100 h-100"
+                        placeholder="share your thoughts"
+                      ></textarea>
```

Comment:
You should include some client-side validation attributes here. Such as `minlength="5" required` for example.
```suggestion
                      <textarea
                        name=""
                        id="user-post-desktop"
                        class="form-control w-100 h-100"
                        placeholder="share your thoughts"
                        minlength="5"
                        required
                      ></textarea>
```